builtin/logical/database/backend.go

```diff
@@ -36,6 +36,8 @@ const (
 	minRootCredRollbackAge  = 1 * time.Minute
 )
 
+var scheduleOptions = cron.Minute | cron.Hour | cron.Dom | cron.Month | cron.Dow
```

Comment:
If possible, I'd suggest moving this to a struct, and having the tests explicitly set it instead of mutating the package level state.

Reply:
Having this as a global var may not work if the tests are being parallelized. You might bump into race conditions if multiple tests are trying to modify this value.

Reply:
Makes sense @calvn
@raymonstah - I think this approach would be a little more involved since `dbPluginInstance` creates an instance per each db config we write , but we don't really need a parser on every one. We decided to go with the different approach so we are able to override a default schedule options (without seconds) in the test.